Tests/xcprojIntegrationTests/XcodeGenTests.swift

```diff
@@ -34,6 +34,14 @@ final class XcodeGenTests: XCTestCase {
         try packageSwiftWithLocalDependency.write(to: packageSwiftPath.url,
                                                   atomically: true,
                                                   encoding: .utf8)
+        // Testing xcodegen as part of xcproj creates a circular dependency; fix up source files affected by API changes
+        let projectGeneratorTestsPath = clonePath + Path("Tests/XcodeGenKitTests/ProjectGeneratorTests.swift")
```

Comment:
I'm sorry that how we test xcproj is not well documented @briantkelley. `XcodeGenTests` is part of a suite of tests, integration tests, that we don't run from PRs. These tests are run after very merge into master to know if there was any change that is breaking. We test that by using XcodeGen and updating the `Package.swift` to point to our changes.
If you revert these changes this PR should pass *(ignore the local failures since they are expected)*

Reply:
Thanks for the info. I saw `swift test` failed and thought I should fix it. But merging it in to the 2.0 branch and updating the framework makes so much more sense.